bar.py

```diff
@@ -3,13 +3,23 @@
 from bar import bar
 from bar import bat
 
+name = None
+
 
 def bar_one(x, y):
     bar.do_x(x)
-    bar.do_y(y)
+    bar.do_y(y + 7)
 
 
 def bar_two(z, q):
+    bat.do_z(z)
+    if name:
+        bat.do_q(q)
+    else:
+        bat.do_q(q + 7)
```

Comment:
**mike bayer** (zzzeek) wrote:

im not sure this part works.

   bat.do_q(q + 9)